lib/ecto/adapters/sql.ex

```diff
@@ -124,6 +126,24 @@ defmodule Ecto.Adapters.SQL do
         :ok
       end
 
+      def do_get_sources(meta) do
```

Comment:
If we can provide such feature, why not?
This not so much LOC, but ability to show all targeted tables is a big plus.